tsconfig.json

```diff
@@ -1,12 +1,17 @@
 {
     "compilerOptions": {
-        "module": "amd",
+        "module": "umd",
```

Comment:
Node requires [special configuration](https://www.npmjs.com/package/node-amd-require) to use amd. So I went with the compromise module configuration. Alternatively we could have a separate tsconfig file for tests to use commonjs module configuration